### PR TITLE
Migrate elements to usage via CDN

### PR DESCRIPTION
**Pull request by siddharthborderwala**
- Fixes the elements crash
- Migrate to using elements via UMD - read about differences and benefits [here](https://leapwallet.notion.site/Integrating-Elements-via-UMD-module-e2234cec91bf48a5a6d304a4bf60d870)
- Use elements UMD types package for TS support

### CODE REVIEW

**Comment by siddharthborderwala** [comment]
@baktun14 I have migrated elements to usage via CDN instead of using as an NPM package. It fixes the crash that's happening due to a Nextjs SWC minification issue.

Some of the other benefits of the UMD build are
- Bundle size of your dApp decreases
- No build/bundling for elements needed

**Comment by baktun14** [comment]
The leap wallet styling overrides the app styling

Console:
image
Console with leap:
image

**Comment by siddharthborderwala** [comment]
@baktun14 thank you for making a note of this. It's a bug with the stylesheet for elements version 1.4.1. We're releasing elements version 1.4.2 today/tomorrow which will have the fix for this. Will update then.

**Comment by siddharthborderwala** [comment]
Hi @baktun14 I have fixed the issue with the stylesheets. It won't affect the rest of the app.

File: apps/deploy-web/src/pages/_document.tsx [hunk 1]
```diff
@@ -25,6 +25,8 @@ export default function MyDocument(props) {
       <body>
         <Main />
         <NextScript />
+        <link rel="stylesheet" href="https://unpkg.com/@leapwallet/elements@1.4.2/dist/style.css" />
+        <script defer src="https://unpkg.com/@leapwallet/elements@1.4.2/dist/umd/main.js" />
```
Comment (baktun14) [inline apps/deploy-web/src/pages/_document.tsx]: I'm not a big fan of importing this globally when we only need it in the get started page.

**Comment by baktun14** [comment]
> Hi @baktun14 I have fixed the issue with the stylesheets. It won't affect the rest of the app.

The styling issues seem to have been fixed. 👍

File: apps/deploy-web/src/pages/_document.tsx (hunk 1 above)
Comment (baktun14) [reply apps/deploy-web/src/pages/_document.tsx]: @siddharthborderwala Would it possible to import these dynamically?

**Review by baktun14 — CHANGES_REQUESTED** [review]

File: apps/deploy-web/src/pages/_document.tsx (hunk 1 above)
Comment (baktun14) [reply apps/deploy-web/src/pages/_document.tsx]: Any update on this? @siddharthborderwala

Comment (siddharthborderwala) [reply apps/deploy-web/src/pages/_document.tsx]: @baktun14 will update this and push changes

Comment (siddharthborderwala) [reply apps/deploy-web/src/pages/_document.tsx]: @baktun14 I've made the required changes. Now the elements related CSS and JS will be loaded only on the `/get-started` page.

**Review by ygrishajev — APPROVED** [review]

**Comment by baktun14** [comment]
@siddharthborderwala Hey, I rebased and fixed some issues, but for some reason the @leapwallet/elements package is still a dependency of the types packages and it messed up our CI...

https://github.com/akash-network/console/actions/runs/10888482586/job/30213077526?pr=317

**Review by baktun14 — APPROVED** [review]

**Pull request merged**